src/jit/compiler.h

```diff
@@ -2825,7 +2825,7 @@ class Compiler
     int gtGetLclVarName(unsigned lclNum, char* buf, unsigned buf_remaining);
     char* gtGetLclVarName(unsigned lclNum);
     void gtDispLclVar(unsigned varNum, bool padForBiggestDisp = true);
-    void gtDispTreeList(GenTree* tree, IndentStack* indentStack = nullptr);
+    void gtDispStmtList(GenTreeStmt* tree, IndentStack* indentStack = nullptr);
```

Comment:
Would be nice to rename `tree` to `stmt`

Reply:
Fixed